spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -192,6 +193,10 @@ private void commitOperation(SnapshotUpdate<?> operation, String description) {
       extraSnapshotMetadata.forEach(operation::set);
     }
 
+    if (!CallerWithCommitMetadata.commitProperties().isEmpty()) {
+      CallerWithCommitMetadata.commitProperties().forEach(operation::set);
+    }
```

Comment:
[question] Should we also add this to SparkPositionWrite
https://github.com/apache/iceberg/blob/b06a89cebd5099b40b188c4c40ea7b1a23d3427a/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java#L250


since starting 3.2 iceberg support MOR with pos deletes

Reply:
Yes, that's a good idea.